Tavisca.Bootcamp.LanguageBasics.Exercise2/Program.cs

```diff
@@ -13,18 +13,78 @@ static void Main(string[] args)
-        {
+        private static void Test(string[] postTimes, string[] showTimes, string expected)
+        {
             var result = GetCurrentTime(postTimes, showTimes).Equals(expected) ? "PASS" : "FAIL";
             var postTimesCsv = string.Join(", ", postTimes);
             var showTimesCsv = string.Join(", ", showTimes);
-            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
+            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
         }
 
-        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
+        {
+           string result= "";
+            //condition if only one exactPostTime is given
+            if(exactPostTime.Length == 1)
+            {
+                result = exactPostTime[0];
+                return result;
+            }
+            //
+            for(int i = 0; i < exactPostTime.Length; i++)
+            {
+                for(int j=0; j<exactPostTime.Length; j++)
+                {
+                    if((exactPostTime[i] == exactPostTime[j]) && (showPostTime[i] != showPostTime[j]))
+                    {
+                        return "impossible";
+                    }
+
+                }
+                var exactHour = exactPostTime[i].Split(":")[0];
+                var HH = int.Parse(exactHour);
+                var exactMinute = exactPostTime[i].Split(":")[1];         
+                var MM = int.Parse(exactMinute);
+                var exactSecond = exactPostTime[i].Split(":")[2];
+                var SS = int.Parse(exactSecond);
+                // if showPostTime contain hour then the total number of hour ago is added to HH which shows current hour 
+                if(showPostTime[i].Contains("hours"))
+                {
+                    var shownHour = showPostTime[i].Split(" ")[0];
+                    HH += int.Parse(shownHour);
+                    if(HH > 24)
+                    {
+                        HH -= 24;
+                    }
+                }
+                // if showPostTime contain minutes then the total number of minutes ago is added to MM which shows current minute 
+                if(showPostTime[i].Contains("minutes"))
+                {
+                    var shownMinute = showPostTime[i].Split(" ")[0];
+                    MM += int.Parse(shownMinute);
+                    if(MM > 59)// if current minute is greater tha 59 then hour is increased buy one 
+                    {
+                        MM -= 60;
+                        HH++;
+                    }
+                    if(HH == 24)//value of HH cannot be 24
+                        HH = 0;
+
+                    HH.ToString();
+                    MM.ToString();
+                    SS.ToString();

```

Comment:
I think this has been mistakenly placed here. 
But if not, ToString() does not modify the data type of the variable(C# is strongly typed), but returns a string. ToString() function has a base implementation(defined in object class) and can be overriden in any implmentation